sla_per_use_system_test.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2016 ScyllaDB
+
+
+import os
+
+from avocado import main
+from sdcm.tester import ClusterTester
+import time
+import yaml
+
+KB = 1024
+
+
+class SlaPerUserTest(ClusterTester):
+
+    """
+    Test SLA per user feature using cassandra-stress.
+
+    :avocado: enable
+    """
+    STRESS_WRITE = 'cassandra-stress write cl=ALL n=1100200300 -port jmx=6868 -mode cql3 native user=cassandra password=cassandra -rate threads=1000'
+    STRESS_READ = 'cassandra-stress read cl=ALL n={n} -port jmx=6868 -mode cql3 native user={user} -rate threads=1000 -pop seq={seq} duration=15m'
+
+    str_pattern = '%8s%16s%10s%14s%16s%12s%12s%14s%16s%16s'
+    ops_threshold_prc = 200
+
+    def __init__(self, *args, **kwargs):
+        super(SlaPerUserTest, self).__init__(*args, **kwargs)
+
+    # Helpers
+    def display_single_result(self, result):
+        self.log.info(self.str_pattern % (result['op rate'],
+                                          result['partition rate'],
+                                          result['row rate'],
+                                          result['latency mean'],
+                                          result['latency median'],
+                                          result['latency 95th percentile'],
+                                          result['latency 99th percentile'],
+                                          result['latency 99.9th percentile'],
+                                          result['Total partitions'],
+                                          result['Total errors']))
+
+    def get_test_xml(self, result, test_name=''):
```

Comment:
looks like it is a copy-paste from `performance_regression_test.PerformanceRegressionTest`